src/main/java/org/mockbukkit/mockbukkit/world/WorldMock.java

```diff
@@ -1887,103 +1890,104 @@ public boolean isGameRule(@Nullable String rule)
 		return this.worldBorder;
 	}
 
+	// Add helper methods:
+	public List<SpawnedParticle> getSpawnedParticles()
+	{
+		return Collections.unmodifiableList(spawnedParticles);
+	}
+
+	public void clearSpawnedParticles()
+	{
+		spawnedParticles.clear();
+	}
+
```

Comment:
Since this methods do not override any method from Bukkit/Paper, can you please include some Javadocs for it please?

Reply:
done